src/main/java/uk/gov/hmcts/befta/exception/InvalidPropertyException.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.befta.exception;
+
+import java.io.IOException;
+
+public class InvalidPropertyException extends IOException {
+
+    /**
+     * Constructs a {@code InvalidPropertyException} with no detail message.
+     */
+    public InvalidPropertyException() {
+    }
+
+    /**
+     * Constructs a {@code InvalidPropertyException} with the
+     * specified detail message.
+     *
+     * @param   msg   the detail message.
```

Comment:
Formatting seems a little off here - 3 spaces surround msg

Reply:
Resolved in https://github.com/hmcts/befta-fw/pull/123/commits/81f37489090fcca5da1dea7fba507fd7e4a6f722